General4.java

```diff
@@ -0,0 +1,16 @@
+public class General4 {
+	public static boolean isPalindrome(String text) {
+		int first = 0;
+		int last = text.length();
+		while(first < last) {
+			if(text.charAt(first) != text.charAt(last))
+				return false;
+			first++;
+			last++;
```

Comment:
Little mistake here. Should be `last--` instead of `++`.

Other than that, good solution & very easy to read